.vscode/extensions.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
You're going to have to remind me what this vscode stuff does

Reply:
This is like a package.json for vscode.  It suggests the appropriate extensions to install for working in this react application (so that they don't need to be listed in the readme.)